src/Misc/layoutbin/en-US/strings.json

```diff
@@ -280,6 +280,7 @@
   "FileNotFound": "File not found: '{0}'",
   "FilePathNotFound": "Cannot find the file path for '{0}'.",
   "FileUploadCancelled": "File upload has been cancelled during upload file: '{0}'.",
+  "FileUploadCurrentFile": "Beginning upload of file: '{0}'",
```

Comment:
we don't loc Debug message.

Reply:
OK, I used line FileContainerServer.cs:241 as inspiration.  I'll remove it.